renderdoc/driver/vulkan/wrappers/vk_resource_funcs.cpp

```diff
@@ -2514,8 +2565,6 @@ VkResult WrappedVulkan::vkCreateImage(VkDevice device, const VkImageCreateInfo *
                                       VK_STRUCTURE_TYPE_EXTERNAL_MEMORY_IMAGE_CREATE_INFO_NV);
           removed |= RemoveNextStruct(&createInfo_adjusted,
                                       VK_STRUCTURE_TYPE_EXTERNAL_MEMORY_IMAGE_CREATE_INFO);
-          removed |=
-              RemoveNextStruct(&createInfo_adjusted, VK_STRUCTURE_TYPE_EXTERNAL_FORMAT_ANDROID);
```

Comment:
Why have you dropped this struct removal? I think we should still be removing this because it is still necessary to get the 'non-external' version - ie. the version we will create on replay - to make sure the memory requirements will be satisfied.